django/dkg/cancer/json_api/endpoints.py

```diff
@@ -0,0 +1,72 @@
+import json # need double quotes in json POST bodies
+from django.http import JsonResponse
+from cancer.models import TrainingArticle, InferenceArticle
+
+
+def train(request):
+    """
+    Receives a POST-request with the following body: ?
+
+    Stores the data only.
```

Comment:
Also alle daten werden im global namespace dann gebacked mit der django db verfuegbar sein oder? 
Oder is das nur um was im function body stehen zu haben?